doc/changelog.rst

```diff
@@ -42,13 +42,19 @@ PyMongo 4.9 brings a number of improvements including:
 - Fixed a bug where PyMongo would raise ``InvalidBSON: date value out of range``
   when using :attr:`~bson.codec_options.DatetimeConversion.DATETIME_CLAMP` or
   :attr:`~bson.codec_options.DatetimeConversion.DATETIME_AUTO` with a non-UTC timezone.
+- The default value for ``connect`` in ``MongoClient`` is changed to ``False`` when running on
+  Function-as-a-service (FaaS).  On some FaaS systems, there is a ``fork()`` operation at function
```

Comment:
Suggest adding `Function-as-a-service (FaaS) like AWS Lambda, Google Cloud Functions, and Microsoft Azure Functions.`

Reply:
Done